### DISCUSSION TITLE
Add Sunshine version in the binaries in Windows and in the localhost webpage

### DISCUSSION THREAD

**Discussion by WilliUreta** [LizardByte/community#127] [Sunshine] [answered]
app.lizardbyte.dev doesn't work when sending a feature request:
` 403 ERROR
The request could not be satisfied.
Request blocked. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.`

Thanks for developing the Sunshine project.
I would like to ask for a new feature. I know that you can run sunshine --version to know which version you have installed, but I think there could be 2 upgrades.
1. Adding the version in the binaries so that "Properties" (at least in Windows) shows what version is running.
2. Somewhere in the localhost webpage, probably using the previous info, showing the version you are running. I would personally add it somewhere in the Home section, maybe under the Hello, Sunshine! title.
Thanks!

**Comment by ReenigneArcher** [comment]
> app.lizardbyte.dev doesn't work when sending a feature request

This might be an issue on your end? We're getting several feature requests there every week, so it's definitely working. Our feedback board has pretty reliable uptime: https://status.lizardbyte.dev/

This is fine here though. In fact I don't know how long I will keep the feedback board on the website.

**Comment by WilliUreta** [comment]
Thanks for the response.
Regarding the version being shown somewhere in the binaries/webcontrol, do you think it's possible or a good feature?

**Reply by ReenigneArcher** [reply]
It's possible and I think a good suggestion.

Another way would be to print the version info first thing and then it would appear at the top of the logs which are now shown in the web UI.

**Comment by brx19** [comment]
Maybe it can be added to the header or footer of the localhost page... On header there is a room on the right side for it.

**Comment by ReenigneArcher** [comment, marked as answer]
This is being added by https://github.com/LizardByte/Sunshine/pull/768
